src/linter/linter.ts

```diff
@@ -10,10 +10,11 @@ import {ProjectGraph} from "@ui5/project";
 import type {AbstractReader, Resource} from "@ui5/fs";
 import ConfigManager, {UI5LintConfigType} from "../utils/ConfigManager.js";
 import {Minimatch} from "minimatch";
+import SharedLanguageService from "./ui5Types/SharedLanguageService.js";
```

Comment:
seems that in this file, a type import would be sufficient?

Reply:
Correct. Thanks. Earlier I created an instance from `lintFile`, but then moved it out to improve the test performance.